front/src/pages/Messages.vue

```diff
@@ -1,81 +1,95 @@
 <template>
   <q-page class="q-pa-md">
     <page-header text="Messages"></page-header>
-    <div class="row q-mt-md">
-      <div class="q-pa-sm" style="width:300px">
-        <search-page v-on:doSearch="search4user"></search-page>
-        <messages-user-list v-bind:messages="subUserList" v-on:selectUser="openMsgThread"></messages-user-list>
-      </div>
-      <div class="q-pa-sm" style="max-width:700px">
-        <messages-conversation v-bind:selUser="selectedUser"></messages-conversation>
+    <div v-if="!userList">
+      <NothingHere :text="text" :image="imgUrl" />
```

Comment:
Please don't replace the entire page, just the active open message and/or the user list. They need to still be able to interact with the page to begin messages.